src/applications/gi-sandbox/containers/search/LocationSearchResults.jsx

```diff
@@ -99,19 +100,62 @@ function LocationSearchResults({
     markers.current.push(markerElement);
   };
 
+  const getCurrentLocationCenter = () => {
+    let center = '';
+    if (streetAddress.searchString) {
+      center = new mapboxgl.LngLat(
+        streetAddress.position.longitude,
+        streetAddress.position.latitude,
+      );
+    }
+    return center;
+  };
+
+  const getZoomLevel = distance => {
+    if (distance === '5') {
+      return 11;
+    } else if (distance === '25') {
+      return 10;
+    } else if (distance === '50') {
+      return 9;
+    } else {
+      return 8;
+    }
+  };
+
   useEffect(
     () => {
       markers.current.forEach(marker => marker.remove());
-
+      const currentmarkerElement = document.createElement('div');
+      currentmarkerElement.className = 'current-position';
       if (!map.current || results.length === 0) return; // wait for map to initialize
-
+      if (streetAddress.searchString) {
```

Comment:
this should be its own function
Why is this not adding the point to the map's bounds like other markers?

Reply:
It's adding the marker to the users location then centering on that location